### ISSUE TITLE
Supporters?  Members?  

### ISSUE THREAD

**Issue by jonathanrobie** [biblicalhumanities/copenhagen-alliance#7]
Do we want 'supporters' or 'members' or both? What do we expect of each? Perhaps 'supporters' simply state their support in public and let us use their logo, 'members' provide resources or people to help us accomplish our mission? 

We need to name and define these roles.

**Comment by jtauber** [comment]
While I agree this distinction _could_ be made, what are benefits and implications of actually making it?

Is the type the choice of the org/individual or does it just get determined automatically by their contribution?

Not against the distinction but want to flesh it out a little more. Are there additional types? e.g. financial support versus support-in-name.

**Comment by rdeblois1960** [comment]
There are many types of contributions: resources, know-how, tools, finances, and sometimes to have only the name + logo on our website can make a significant difference.

**Comment by jonathanrobie** [comment]
I care mostly about two things:

- Can we use your logo or your name in an endorsement on our web site and other marketing materials?
- Should you be included in a vote?  If we have multiple members from the same institution, do we need to balance that somehow?  (This is less crucial with a consensus-driven approach to counting votes, but still worth considering.)

**Comment by ernstboogert** [comment]
In my opinion, institutions exist only because of their members. My proposal would be to connect the Copenhagen Alliance with members. Since CA-members are also members of their own institutions, it fits perfectly to visualize this affiliation when CA-members are mentioned. 

Moreover, if certain datasets, tools, etc. are made by people connected to those institutions, it is appropriate to give credits and to connect their institution's logo to a developed dataset, tool, etc.

But, what does it mean to connect institutions to the CA alliance, apart from their affiliated members? In my view, it is not the institutions that are allied at the moment, but we as to certain institutions affiliated members. This, of course, poses the question whether that has to change in the near future... 

By the way, what does it mean to users of our website to have 25 logos of supporting institutions on the screen? Or what does it mean to have 10 logo's on a pamphlet? I think the CA has a great job to show the world its own quality. Clearly, a clear presentation of the contributors and their affiliations will help much. Although products of the CA should 'sell' well in the future because of its known (and shown) quality, adhered logos can help if appropriate. 

The backside is that work done by nonacademically affiliated people can have the smell of being unacademic, especially when an academic logo lacks...

For me, the most important point is to give much attention to quality assurance and quality checking, to give _materials provided by the CA_ a value on its own. Message: we should give most attention to our own logo and the importance of that sign.

**Comment by jonathanrobie** [comment]
I think we may have closed this in passing when we resolved #4 - here is the description we adopted:

> The Copenhagen Alliance for Open Biblical Language Resources is a diverse coalition of organizations, institutions, and individuals with a common interest in making biblical language data free and openly accessible to anyone for research, language learning, translation, and other uses.

We can simply refer to organizations, institutions, and individuals without specifying terms like members or supporters, at least for now.  If we have to get more specific later, we can cross that bridge then.

**Comment by jcuenod** [comment]
There still seems to be a question of membership in my mind. How does an interested party (organizations, institutions, or individual) get involved?

**Comment by jtauber** [comment]
Yeah, I'd reopen. We haven't resolved whether we have a distinction between 'supporters' and 'members' and what the roles are. And @jcuenod's question: How does an interested party (organizations, institutions, or individual) get involved?

**Comment by jonathanrobie** [comment]
OK, reopening.  Incidentally, feel free to reopen an issue yourself if you feel I have closed prematurely.  Even if you don't know if anyone else agrees with you yet.  We can then see how much agreement we have.

**Comment by jtauber** [comment]
I couldn't see how to open it myself.

**Comment by jonathanrobie** [comment]
This issue is on our critical path.  Before announcing the coalition more broadly, we need to be able to say who is behind it.

**Comment by jonathanrobie** [comment]
I have discussed this with several people, here are some thoughts:

I think membership should require at least:
- Agreeing to our mission.
- Agreeing to be mentioned as members on our website.
- Freely releasing at least one resource (for producers) or showing at least one convincing and freely licensed application using multiple resources together (for consumers).

We probably need more than one kind of membership to allow for research institutions, Bible translation associations, and individuals to join.  The requirements might vary.  

Some members may do most of the work, others may do less work creating resources but lend credibility by endorsing us, some may work primarily in research settings, others in other settings.  I think all of that is OK.

**Comment by waldeinburg** [comment]
I find Jonathan's proposal for membership requirements perfectly sensible. However, no. 3 means that I currently don't qualify as a member because I haven't produced anything yet. So a chance to stay in touch as "supporter" would be nice.

**Comment by jtauber** [comment]
On @waldeinburg's point, I wonder if we need a "supporter" (your first two bullet points, @jonathanrobie) vs "contributor" (all three bullet points) distinction.

_Supporters_ (or whatever we call them) put their name behind the alliance, _Contributors_ (or whatever we call them) additional contribute resources openly. Perhaps only the latter gets a vote.
